draft-irtf-cfrg-vdaf.md

```diff
@@ -4508,12 +4507,11 @@ picks bogus measurements for the remaining Clients.  Applications can guard
 against these risks by adding additional controls on report submission,
 such as Client authentication and rate limits.
 
-VDAFs do not inherently provide differential privacy {{Dwo06}}.  The VDAF approach
-to private measurement can be viewed as complementary to differential privacy,
-relying on non-collusion instead of statistical noise to protect the privacy of
-the inputs.  It is possible that a future VDAF could incorporate differential
-privacy features, e.g., by injecting noise before the sharding stage and
-removing it after unsharding.
+VDAFs do not inherently provide differential privacy {{Dwo06}}: depending on
+the distribution of the measurements, the aggregate result itself can still
+leak a significant amount of information about the measurements. Thus
+applications SHOULD execute a VDAF with a compatible mechanism for differential
```

Comment:
How does an implementer know what makes a mechanism compatible?

When is this a MUST?

Reply:
> How does an implementer know what makes a mechanism compatible?

I don't think we can answer this question without being concrete about what DP mechanisms are available. There is an I-D for this, but its in early stages and doesn't have a home at IETF yet: https://github.com/wangshan/draft-wang-ppm-differential-privacy/

> When is this a MUST?

It really depends on the application: for Poplar1, we think it's always going to be necessary to stop attacks (https://github.com/cfrg/draft-irtf-cfrg-vdaf/pull/336); for Prio3, it doesn't seem strictly necessary in all cases.

Reply:
For the record, @ekr suggests on list that a SHOULD is inappropriate and the sentence should just be deleted.

Reply:
Might I suggest one of the additional options in RFC 6919, e.g., [SHOULD CONSIDER](https://rfcs.online/rfcs/rfc6919.html#section-2)?

More seriously, the idea of composition seems worth mentioning, but I agree that RFC 2119 language probably isn't appropriate.

Reply:
@martinthomson also concurs on list, I'll remove it!